getting_started_scenarios/ssm_scenario/Readme.md

```diff
@@ -0,0 +1,42 @@
+# AWS Systems Management (SSM) Getting Started Scenario
```

Comment:
SSM is not an approved abbreviation. Check the internal offering names page and fix this throughout this PR (I've seen it used in most files so be sure to check them all).